src/Microsoft.Azure.WebJobs.Extensions.EventHubs/Listeners/EventHubListener.cs

```diff
@@ -88,9 +91,11 @@ public async Task StopAsync(CancellationToken cancellationToken)
             _started = false;

```

Comment:
Is there any possibility of StopAsync() getting called concurrently ?  Not sure what happens if UnregisterEventProcessorAsync() is called concurrently.  We may want to add a lock around line this call.